### ISSUE TITLE
[Bug]: Invalid destination when making payment to LN only payment request

### ISSUE THREAD

**Issue by jacobburrell** [btcpayserver/btcpayserver#5704]
### What is your BTCPay version?

© BTCPay Server v1.12.3+f23078df1

### How did you deploy BTCPay Server?

[Lunanode](https://lunanode.com/)

### What happened?

When attempting to pay to any LN invoice, with and without amount, short or long exp. date, etc.

<img width="1383" alt="image" src="https://github.com/btcpayserver/btcpayserver/assets/61520543/aa3e811b-b448-472b-8c2f-4607b189190b">

Invalid destination or payment method

### How did you encounter this bug?

1. Made payment request
2. Made LN invoice
3. Specified invoice from #2 and attempted to send full amount, partial amount, even 1 satoshi amounts.

### What browser do you use?

Google Chrome | 121.0.6167.85 (Official Build) (x86_64)

-- | --
Revision | 3f98d690ad7e59242ef110144c757b2ac4eef1a2-refs/branch-heads/6167@{#1539}
OS | macOS Version 14.2.1 (Build 23C71)

### Additional information

I can directly make a payment to the same invoice from my LN node without issue from other interfaces.

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by pavlenex** [comment]
@Kukks Any idea what may be wrong here? The error message isn't quite clear.

**Comment by jacobburrell** [comment]
Let me know what logs might be useful to add here too. It doesn't seem like an issue with my LN node, but I don't see anything relevant at /server/logs

I haven't looked at the docker containers logs yet.

**Comment by pcwd** [comment]
Getting this problem as well.  Click Refund on a paid invoice.  Pull Payment is created. Invoice is posted and get this error.  In the invoice view, the payment is marked as refunded and the lightning invoice has been recorded there.  However, no Payout Request was created to be approved.  I had setup automated Payouts and thought that may have been the problem.  But, when I turned that off so that I had to manually approve payouts, sill no payout request is being created.  But the invoice for the payout request is still recorded on the invoice record as a paid refund. 

tl;dr  Problem seems to be with an error creating the Payment request, rather than a problem with the invoice format as the error suggests.

**Comment by pavlenex** [comment]
@jacobburrell are you still experiencing this issue?
